common/pkcs11.h

```diff
@@ -754,6 +809,25 @@ struct ck_mechanism_info
   ck_flags_t flags;
 };
 
+typedef unsigned long ck_param_type;
+
+typedef struct ck_otp_param {
+   ck_param_type type;
+   void *pValue;
+   unsigned long ulValueLen;
```

Comment:
For consistency with the GNU convention, I would suggest to rename members in snake case without type prefix: ulValueLen -> value_len.